src/lib/components/pill.svelte

```diff
@@ -22,15 +22,15 @@
 
 <style lang="postcss">
   .pill {
-    @apply flex items-center justify-center gap-1 rounded-full border-2 border-slate-100 px-3 py-1 text-sm;
+    @apply flex items-center justify-center gap-1 rounded-full border-2 border-slate-50 px-3 py-1  text-sm text-black text-inverse;
   }
 
   .gray {
-    @apply bg-badge text-slate-600;
+    @apply border-none bg-transparent text-primary;
```

Comment:
Small suggestion for the text color here

```suggestion
    @apply bg-badge text-disabled;
```
<img width="245" alt="Screenshot 2024-04-11 at 5 09 47 PM" src="https://github.com/temporalio/ui/assets/15069288/8c23d7ce-d439-42a6-b0d0-e0882a9056d1">